spec/components/shared/navbar_component_spec.rb

```diff
@@ -5,6 +5,7 @@
 
   before do
     allow(controller).to receive(:jobseeker_signed_in?).and_return(true)
+    allow(controller).to receive(:publisher_signed_in?).and_return(false)
```

Comment:
Why is this needed?

Reply:
because otherwise we have errors with devise as it sets things that are not accessible from view components